test/system/tools/fastdds/tests.py

```diff
@@ -96,6 +96,13 @@ def test_fastdds_installed(install_path):
         print('test_fastdds_installed FAILED')
         sys.exit(ret)
 
+def test_fastdds_version(install_path):
+    """Test that fastdds version is printed correctly."""
+    args = ' -v'
+    ret = subprocess.call(cmd(install_path, args=args), shell=True)
+    if 0 != ret:
+        print('test_fastdds_version FAILED')
+        sys.exit(ret)
```

Comment:
We need to add the test to the `test/system/tools/fastdds/CMakeLists.txt` in order to build it and be able to run it. Remember also to add it to the `Tests dictionary` at the end of the file.